pkg/network/usm/usm_http2_monitor_test.go

```diff
@@ -492,21 +492,23 @@ func (s *usmHTTP2Suite) TestRawTraffic() {
 	tests := []struct {
 		name              string
 		skip              bool
-		messageBuilder    func() []byte
+		messageBuilder    func() [][]byte
```

Comment:
❓ maybe we should create a different test for those type of tests which contains a message builder of type [][]byte, it is not necessary for the rest of the (current tests) while it can be really useful for the other tests which will be added

Reply:
I think it will be great for the tests you're about to add, and there is a minimal "impact" on the other tests